lib/Migration/Version000004Date20210331142734.php

```diff
@@ -34,68 +35,68 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 
 		if (!$schema->hasTable('approval_rules')) {
 			$table = $schema->createTable('approval_rules');
-			$table->addColumn('id', 'integer', [
+			$table->addColumn('id', Types::INTEGER, [
 				'autoincrement' => true,
 				'notnull' => true,
 				'length' => 4,
 			]);
-			$table->addColumn('tag_pending', 'integer', [
+			$table->addColumn('tag_pending', Types::INTEGER, [
 				'notnull' => true,
 				'length' => 4,
 			]);
-			$table->addColumn('tag_approved', 'integer', [
+			$table->addColumn('tag_approved', Types::INTEGER, [
 				'notnull' => true,
 				'length' => 4,
 			]);
-			$table->addColumn('tag_rejected', 'integer', [
+			$table->addColumn('tag_rejected', Types::INTEGER, [
 				'notnull' => true,
 				'length' => 4,
 			]);
-			$table->setPrimaryKey(['id']);
+			$table->setPrimaryKey(['id'], 'appr_rule_id');
```

Comment:
if you change this in retrospective it means existing installations will have a different primary key index name than new installations. That is not a problem by itself. But if you every reference `appr_rule_id` you have to remember that it might or might not exist.

Reply:
Right, let's not change the index names.